website/docs/cdktf/index.html.md

```diff
@@ -8,8 +8,7 @@ description: |-
 
 # What is CDK for Terraform?
 
-Cloud Development Kit for Terraform (CDKTF) allows you to use familiar
-programming languages to define and provision infrastructure. This gives you access to the entire Terraform ecosystem without learning HashiCorp Configuration Language (HCL) and lets you leverage the power of your existing toolchain for testing, dependency management, etc.
+Cloud Development Kit for Terraform (CDKTF) allows you to use familiar programming languages to define and provision infrastructure. This gives you access to the entire Terraform ecosystem without learning HashiCorp Configuration Language (HCL) and lets you leverage the power of your existing toolchain for testing, dependency management, etc.
 
 We currently support TypeScript, Python, Java, C#, and Go (experimental).
 
```

Comment:
I get a 404 when I try to access this terraform platform image https://github.com/hashicorp/terraform-cdk/blob/main/website/docs/cdktf/terraform-platform.png
maybe it hasn't been moved from it's prior location yet? 
@laurapacilio

Reply:
oh! For a second I was stumped, but you know what it is? The path you sent is on main! and this image is definitely not in that location on main. Try this: https://github.com/hashicorp/terraform-cdk/blob/terraform-io-docs/website/docs/cdktf/terraform-platform.png